Dispatcher improvements

akodanka opened the pull request:
Improvements for remote inference using dispatcher.

rairatne committed:
Compatibility changes to build nn-hal for Android R

Tracked-On: OAM-105850
Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>
Signed-off-by: Ratnesh Kumar Rai <ratnesh.kumar.rai@intel.com>
Signed-off-by: Jaikrishna, Nemallapudi <nemallapudi.jaikrishna@intel.com>

akodanka committed:
Disable parallel attempts for remote inference

Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>

rairatne committed:
Added load model and data_type for remote infer

loadmodel rpc call added after sending IR files
Included data_type parameter for input data

Tracked-On: OAM-109729
Signed-off-by: Ratnesh Kumar Rai <ratnesh.kumar.rai@intel.com>

akodanka committed:
Enable debug logs

Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>

akodanka committed:
Increase GRPC message size and improved remote checks

Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

akodanka committed:
Disable remote inference if a request fails

Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>

akodanka committed:
Parallel Remote Inference

Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

sysopenci commented:
Expected syntax:
    Commit message one-line summary

      Extra paragraphs can be added here.

      Longer description that explains the problem / solution
      or whatever that needs to be said

      It can contain
        multiple lines
        multiple paragraphs
        blank lines
        references

      But each paragraph must be followed by a blank line

      However, you need to keep the 'Tracked-On', 'Change-Id',
      'Signed-off-by', ..., fields in the **lastest** paragraph of
      the commit message.

      Change-Id: I9e58f79c7faf7f0edb9792e8b5322ab161331141
      Tracked-On: https://jira.server.ext/browse/PROJECTA-123
      Signed-off-by: Developer Name <developer.name@intel.com>

akodanka committed:
Prevent non-compliant upcast of model, operand

Fix the following errors :
Upcasting non-compliant model
Upcasting non-compliant operand type TENSOR_QUANT8_ASYMM_SIGNED from V1_3::OperandType to V1_2::OperandType

Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>

rairatne committed:
Resolved quant type conversion issue

Signed-off-by: Ratnesh Kumar Rai <ratnesh.kumar.rai@intel.com>

akodanka committed:
Background remote model load

Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>

akodanka committed:
Separate ModelInfo objects for each operation

Signed-off-by: Anoob Anto K <anoob.anto.kodankandath@intel.com>
